packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -494,38 +495,47 @@ const PlanStep: React.FunctionComponent< PlanStepProps > = ( {
 type StepIndexRenderFunction = ( renderOptions: {
 	stepIndex: number;
 	forwardStepIndex: boolean;
-} ) => ReactNode;
+} ) => React.ReactNode;
 
 const Summary: React.FunctionComponent = () => {
-	const { title, updateTitle, saveTitle, isSiteTitleStepVisible, showSiteTitleStep } = useTitle();
+	const hasSelectedDomain = useSelect( ( select ) => select( LAUNCH_STORE ).hasSelectedDomain() );
+	const selectedDomain = useSelect( ( select ) => select( LAUNCH_STORE ).getSelectedDomain() );
+	const selectedPlan = useSelect( ( select ) => select( LAUNCH_STORE ).getSelectedPlan() );
```

Comment:
Not sure if this bug was introduced by this PR, but if the site already has an associated premium plan, then `selectedPlan` seems to be `undefined`.

This causes 2 main issues:
- The disabled plan item at the top of the Summary view has an empty label

![image](https://user-images.githubusercontent.com/1083581/100133448-10aa7000-2e87-11eb-9a21-c5788ba3dc07.png)

- The "Launch your site" button is always `disabled`

Reply:
This is an existing issue (we need to open a GH ticket):
> The disabled plan item at the top of the Summary view has an empty label

Thanks for spotting the disabled button. Added a new item to the ToDo list in the description for it

Reply:
> This is an existing issue (we need to open a GH ticket):

I opened https://github.com/Automattic/wp-calypso/issues/47734 to track this bug.

I believe there's a chance that fixing the bug afflicting the disabled "Launch your site" button will also fix this new issue.

Reply:
Fixed the missing purchased plan label in https://github.com/Automattic/wp-calypso/pull/47601/commits/1a559a45892290b88d4880ab14e360550c172c08

Context: when https://github.com/Automattic/wp-calypso/pull/47378 was approved, the corresponding diff (D52710-code) wasn't yet deployed and finally it was abandoned in favour of a frontend solution.

Reply:
Great find! Thank you for looking into this issue

Reply:
Fixed also the second issue mentioned above (launch button being always disabled) in https://github.com/Automattic/wp-calypso/pull/47601/commits/f749acab90d87d8be50b42f772a89df02d7674d4